Source/WebInspectorUI/UserInterface/Views/LocalResourceOverridePopover.js

```diff
@@ -115,6 +121,19 @@ WI.LocalResourceOverridePopover = class LocalResourceOverridePopover extends WI.
                 return null;
 
             data.responseHeaders = headers;
+
+            data.mappedFilePath = this._mappedFilePathValueElement.textContent;
```

Comment:
Understood and agreed. The suggestion was just a nit / preference. 

The argument for using an `<input type=hidden>` _in addition to_ `this._mappedFilePathValueElement.textContent` was to meet the expectation that the serialized data to configure the `LocalResourceOverride` is collected from form fields.

It confused me for a bit that the output from the file picker didn't _appear_ to be used anywhere, yet the feature did work (that's because I didn't expect `this. _mappedFilePathValueElement.textContent` to be used for anything other than display).